### ISSUE TITLE
Error converter usageEventLogs/usage_events  

### ISSUE THREAD

**Issue by Mathewwss** [pkp/pkp-lib#9489]
Update OJS 3.3.0.14 to 3.4.0.3.

Update guide : https://docs.pkp.sfu.ca/dev/upgrade-guide/en/

![image](https://github.com/pkp/pkp-lib/assets/50458770/e81fa971-e516-48f0-b1e7-caed356598b5)

Do not configure "files" and "public" in config.inc.php to update completed successfully. And configure after update.

Is it incorrect to do it this way???

**Comment by asmecher** [comment]
@Mathewwss, could you ask this question on our support forum (https://forum.pkp.sfu.ca) rather than filing an issue? That's the best place to get support.

**Comment by bozana** [comment]
@Mathewwss, once you write a question/post in the forum, can you address me (@bozana) there, so I can take a look -- the files folder should be there and have correct path...

**Comment by Mathewwss** [comment]
I didn't post it on the forum, because I found a similar post.

I don't remember the URL of the post, I'll look again if I find it and leave it here in the next message.

Solved the problem by changing the IP of the requests and the website to the IP of the host machine where OJS is installed.

Original:

172.16.16.5 - - "2023-01-02 00:04:25" http://url/index 200 "Mozilla/5.0 (X11; U; Linux; en-US) AppleWebKit/527  (KHTML, like Gecko, Safari/419.3) Arora/0.10.1"

Change (OJS machine = 172.16.16.20):

**172.16.16.20** - - "2023-01-02 00:04:25" http://**172.16.16.20**/index 200 "Mozilla/5.0 (X11; U; Linux; en-US) AppleWebKit/527  (KHTML, like Gecko, Safari/419.3) Arora/0.10.1"

New alert: 

There was a problem finding IP in the Geo database at /var/www/ojs-files/usageStats/IPGeoDB.mmdb. Error: The address 172.16.16.20 is not in the database.


Despite the alert, the update was completed.


Any problem doing it this way ?
